src/pages/en/guides/data-fetching.md

```diff
@@ -171,3 +171,42 @@ const data = await response.json();
 ```
 
 See the full tutorial [Building an Astro Website with WordPress as a Headless CMS](https://blog.openreplay.com/building-an-astro-website-with-wordpress-as-a-headless-cms) to add WordPress to your Astro Project! 
+
+### Example: Crystallize
+
+```astro
+---
+// src/pages/about.astro
+// Fetch Shop's navigation based on Crystallize's tree.
+
+import BaseLayout from '../../layouts/BaseLayout.astro';
+import { createClient, createNavigationFetcher } from '@crystallize/js-api-client';
+
+const apiClient = createClient({
+    tenantIdentifier: 'furniture'
+});
+
+const fetch = createNavigationFetcher(apiClient).byFolders;
+const navigation = await fetch('/shop', 'en', 2);
+
+// or Fetch anything directly from the client
+const response = await apiClient.catalogueApi(`
+  query {
+    tenant {
+      name
+    }
+  }
+`);
+
+---
+<BaseLayout>
+  <h1>{response.tenant.name}</h1>
+	<nav>
+		<ul>
+      {navigation.tree.children.map(child => (
+        <li><a href={child.path}>{child.name}</a></li>
+      ))}
+		</ul>
```

Comment:
I was thinking that maybe we can keep the data fetching as short as possible to match other examples. I took the request example from https://crystallize.com/learn/developer-guides/catalogue-api/querying-the-catalogue

```suggestion
```astro title="src/pages/index.astro"
---
// Fetch your catalogue paths from Crystallize graphql API

import BaseLayout from '../../layouts/BaseLayout.astro';
import { createClient } from '@crystallize/js-api-client';

const apiClient = createClient({
    tenantIdentifier: 'furniture'
});

const query = `
  query getCataloguePaths{
    catalogue(language: "en", path: "/shop") {
      name
      children {
        name
        path
      }
    }
  }
`
const { data } = await apiClient.catalogueApi(query)
---
<BaseLayout>
  <h1>{catalogue.name}</h1>
	<nav>
		<ul>
      {catalogue.children.map(child => (
        <li><a href={child.path}>{child.name}</a></li>
      ))}
		</ul>
```

Reply:
it is fine with me. I just wanted to show that the client as some helpers versus simple GraphQL query. Bu that's totally fine.